include/RAJA/policy/sequential/forall.hpp

```diff
@@ -104,17 +104,17 @@ RAJA_INLINE void forall(const PolicyBase &, Iterable &&iter, Func &&loop_body)
   }
 }
 
-template <typename Iterable, typename Func>
+template <typename Iterable, typename Func, typename IndexType>
```

Comment:
Is having template parameter names shadow typedefs a bad idea? I remember this happened in the type traits header....